### ISSUE TITLE
[Fabric 1.20.1] Bronze recipe gives 2 ingots instead of 3, Ore-to-ingot bronze recipe has similar issues too.

### ISSUE THREAD

**Issue by v3xed-v0id** [Noaaan/MythicMetals#270]
Bronze metal requires 2 copper ingots and 1 tin ingot in recipe, yet only gives 2 bronze output, it should give the same amount of output as there is for input. which should be 3.

Similarly, the direct ore-to-ingot bronze recipe also only gives 2 output, it should give 3 ingot at the very least (tho its inefficient due to loss since ore-to-ingot for pure metal is 2-to-3, and this hypothetical ratio would be 3-to-3, maybe make the direct ore recipe make 4 ingot instead? 4 ingot would still be a slight inefficiency but its probably about as balanced as you could get it to be)

Please consider, thank you for reading.

**Comment by v3xed-v0id** [comment]
I will add on to my main post, that there is an argument for removing the direct ore-to-ingot recipe for bronze and forcing players to refine their main pure ingots before making alloys, it would also remove the inbalance issues there is with the current recipe and make the process more straightforward and refined. Tho this is just one of the many paths there is to address this current issue.

But yeah, the main issue is the ingot-to-ingot bronze recipe which has no real reason to make only 2 when it should be making 3 IMO cus it uses 3 ingots to make.

**Comment by Noaaan** [comment]
I agree that the Bronze Alloy Ingot recipe could use a small increase, yeah. Removing/changing the Raw Ore recipe is out of the picture - both with Fortune and higher tiers of forges the yield is better than smelting via ingots first (note that this varies per metal)
